cheatsheets/Threat_Modeling_Cheat_Sheet.md

```diff
@@ -2,52 +2,54 @@
+- Document as many potential threats to the system as possible.
+- Document security controls that may be put in place to reduce the likelihood or impact
+  of a potential threat.
 
-Following the guidance in this cheat sheet, the assessors will list all possible risks and then verifies whether there are enough security controls to protect against these risks. The assessor will then give better recommendations on how to mitigate these risks. It will help the assessor discover logical attacks. In general, the threat modeling will help designers, architects and assessors discover logical attacks.
+Note that throughout the document, the terms "systems" and "applications" are used interchangeably. The principles in the document apply equally to designing and building systems such as network infrastructures or server clusters as they do to designing or developing desktop, mobile, or web applications.
 
-## Preparation
+### Threat Modeling Terminology
 
-### Understand Risk Management Basics in the context of Application Security
+You should be familiar with the following terms that will be used throughout this cheat sheet.
 
-Understand the Relation between Risk, Threats, and Vulnerabilities.
+A *threat agent* is an individual or group that is capable of carrying out a particular threat. It is fundamental to identify who would want to exploit the assets of a company, how they might use them against the company, and if they would be capable of doing so. Some threats require more expertise or resources, and thus raise the level of threat actor needed. For example, if a threat requires hundreds of thousands of dollars of computing power to implement, it is likely that only organized corporate, criminal, or government actors would be valid threat actors for such a threat. However, with the rise of cloud computing and the prevalence of attack software on the internet, other threats may be easy to implement with relatively little skill and few resources.
 
-### Threat Modeling Terminologies
+*Impact* is a measure of the potential damage caused by a particular threat. Impact and damage can take a variety of forms. A threat may result in damage to physical assets, or may result in obvious financial loss. Indirect loss may also result from an attack, and needs to be considered as part of the impact. For example, if your company's website were defaced this could cause damage to your company's reputation, which may in turn cause a loss of business because of the loss of confidence by your users. Depending on the business you are in, attacks that expose user information could potentially result in a physical threat of harm or loss of life to your users, greatly raising the impact of threats that would allow such exposure.
 
-*Information Asset*, a body of knowledge that is organized and managed as a single entity. Like any other corporate asset, an organization's information assets have financial value.
```

Comment:
Some of the terms in this heading weren't even used elsewhere in the cheat sheet, so it seemed superfluous to define them.